framework/src/modules/poa/constants.ts

```diff
@@ -34,6 +34,10 @@ export const COMMAND_UPDATE_KEY = 'updateKey';
 export const COMMAND_UPDATE_AUTHORITY = 'updateAuthority';
 export const MAX_UINT64 = BigInt(2) ** BigInt(64) - BigInt(1);
 export const LENGTH_PROOF_OF_POSESSION = 96;
+export const defaultConfig = {
+	authorityRegistrationFee: AUTHORITY_REGISTRATION_FEE.toString(),
+};
+
```

Comment:
we get the JSON format from the config so all the bigint and buffer values are coming as string values and are converted later into specific types

Reply:
Since we are adding registration fee to the config here, it might make sense to update the `getRegistrationFee` endpoint to get the fee from the config instead of from constants directly.